deployer/tasks/setup.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Exception\RuntimeException;
+use Deployer\Task\Context;
+use Symfony\Component\Yaml\Yaml;
+
+desc('Setup redaxo instance');
+task('setup', new class {
+    public function __invoke()
+    {
+        $this->checkConfigYml();
+    }
+
+    private function checkConfigYml()
+    {
+        cd('{{release_path}}');
+
+        if (test('[ -f {{data_dir}}/core/config.yml ]')) {
+            return;
+        }
+
+        writeln('');
+        writeln('<comment>Setting config.yml</comment>');
+        writeln('');
+
+        $config = Yaml::parse(file_get_contents(getcwd().'/'.get('data_dir').'/core/config.yml'));
+
+        $config['setup'] = false;
+        $config['debug'] = false;
+        $config['instname'] = 'rex'.date('YmdHis');
+
+        $config['server'] = ask('Server:', 'https://'.Context::get()->getHost()->getRealHostname());
+        $config['servername'] = ask('Server name:', $config['servername']);
+        $config['error_email'] = ask('Error email:', $config['error_email']);
+
+        writeln('');
+
+        $db = $config['db'][1];
+        $db['host'] = 'localhost';
```

Comment:
die defaults würd ich nur nehmen, wenn in der config.yml nicht schon werte drinnen standen.. ansonsten die werte aus der datei als default nehmen

Reply:
Ich gehe aus von der lokalen config.yml. Dort steht also die lokale DB drin, daher macht es keinen Sinn meiner Meinung nach, die als default zu nehmen (solange man es bei der lokalen config.yml als Basis belässt).
Eine config.yml dort gibt es ja bisher nicht, siehe Test weiter oben.